tests/reject-tests.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+# Copyright (C) 2014 Nikos Mavrogiannopoulos
+#
+# License: BSD
+
+srcdir="${srcdir:-.}"
+
+echo "***********************************************"
+echo "This test will use a radius server on localhost"
+echo "and which can be executed with run-server.sh.  "
+echo "Test passes invalid client credentials and     "
+echo "expects it to be rejected by Server.           "
+echo "***********************************************"
+
+TMPFILE=tmp$$.out
+
+if test -z "$SERVER_IP";then
+	echo "the variable SERVER_IP is not defined"
+	exit 77
+fi
+
+PID=$$
+sed -e 's/localhost/'$SERVER_IP'/g' -e 's/servers-temp/servers-temp'$PID'/g' <$srcdir/radiusclient.conf >radiusclient-temp$PID.conf
+sed 's/localhost/'$SERVER_IP'/g' <$srcdir/servers >servers-temp$PID
+
```

Comment:
What are you actually testing here? A reject in general, or a reject due to an EAP server?